tools/copy_bom/.gitignore

```diff
@@ -0,0 +1,9 @@
+/target
+run*
+Cargo.lock
```

Comment:
Should not ignore the Cargo.lock file.
https://github.com/rust-lang/cargo/issues/315

Reply:
fixed